addons/mail/static/src/models/message_tracking_value_item.js

```diff
@@ -0,0 +1,101 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+import { format } from 'web.field_utils';
```

Comment:
Sounds like legacy stuff. You may want to use functions from `@web/fields/formatters` instead

Reply:
This was how it goes and I searched for the new formatter, only a few types are supported. like Date, Datetime. Char are not supported. So I prefer to remain the code?

Reply:
Having a formatter for fields of type Char doesn't seem really useful, this is probably the reason why there is none, I guess. I think it's better not to put legacy stuff back in rewritten code, so I'd encourage you to use formatters from `@web/fields/formatters` instead.

Note that without the legacy formatter, the case when the field of type Char is `False` might require to be treated as a special case.

Reply:
format.char is taking care of type `char` `many2one` and `selection`. I think that's why they are using this actually.